src/apps/core/submodules/alerts/alerts.js

```diff
@@ -258,6 +305,9 @@ define(function(require) {
 				}).map(function(alerts, type) {
 					return {
 						type: type,
+						unreadCount: _.reduce(alerts, function(total, alert) {
```

Comment:
Just found that `lodash#sumBy` can be used in this case, to make it a little bit shorter:

```
unreadCount: _.sumBy(alerts, function(alert) {
    return _.includes(readAlertIds, alert.id) ? 0 : 1;
}),
```